src/components/SingleArticleManager.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react'
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+import SingleArticleCard from './SingleArticleCard';
+
+export default function SingleArticleManager({activeUserName}){
+    const [voteArticle, setVoteArticle] = useState(null)
+    const [commentId, setCommentId] = useState(null)
+    const [commentsVote, setCommentsVote] = useState(null)
+    
+    const [article, setArticle] = useState({})
+
+    const {id}  = useParams();
+    useEffect(()=>{
```

Comment:
Consider extracting your api calls out into their own file for cleaner code and to make them reusable in the future should you need them elsewhere.